doc/source/release/1.23.1-notes.rst

```diff
@@ -4,42 +4,42 @@
 NumPy 1.23.1 Release Notes
 ==========================
 
+The NumPy 1.23.1 is a maintenance release that fixes bugs discovered after the
+1.23.0 release. Notable fixes are:
 
-Highlights
-==========
+- Fix sorting of float16 NaNs 
```

Comment:
Oh, randomly realized... The PR title is a bit misleading, since this is actually searchsorted not sorting.  But, I think its close enough that nobody will care or even notice :).

Reply:
Hmm, I can fix that up.